bin/generate-changelog.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -o errexit -o errtrace -o nounset -o pipefail
+
+: ${GITHUB_CHANGELOG_GENERATOR:=github_changelog_generator}
+: ${GEM:=gem}
+
+generate_changelog() {
+    local -r sVersion="${1?One parameter required: <release-to-generate>}"
+
+    if ! command -v "${GITHUB_CHANGELOG_GENERATOR}" >/dev/null 2>&1;then
+        echo "This script requires the '${GITHUB_CHANGELOG_GENERATOR}' Ruby Gem"
+
+        if ! command -v "${GEM}" >/dev/null 2>&1;then
+            echo "Could not find the '${GEM}' command needed to install 'github_changelog_generator'!" >&2
+            echo 'Aborting.'
+            exit 67
+        else
+            echo "Installing '${GITHUB_CHANGELOG_GENERATOR}'..."
+            gem install github_changelog_generator
+        fi
+    fi
+
+    local -r sChangelog="$(
+        "${GITHUB_CHANGELOG_GENERATOR}"                 \
+            --user Dealerdirect                         \
+            --project phpcodesniffer-composer-installer \
+            --token "$(cat ~/.github-token)"            \
```

Comment:
Is this a local file ? If so, should the need for this file, what it is and how to create it, be mentioned somewhere ?

Reply:
That shouldn't be there at all. I'll add it as a parameter to be passed to the script. :+1: